code/Terrain/TerrainVertex.cs

```diff
@@ -0,0 +1,32 @@
+﻿#region Assembly Sandbox.Game, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null
+// C:\Program Files (x86)\Steam\steamapps\common\sbox\bin\managed\Sandbox.Game.dll
+#endregion
```

Comment:
if we are ripping things from the s&box DLLs maybe we should make it not obvious